Documentation/Doxygen/General/src/work_with_examples.md

```diff
@@ -178,7 +178,11 @@ The settings of the *csolution project* are in the `*.yml` files. Adjust these s
   ![Typical Compiler Options Settings](Options-C.png)
 
 - *Linker*: Configure Scatter File and adjust warnings.
-  - The native uVision project manager does not offer the same [linker script management](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/build-overview.md#linker-script-management). Copy therefore the preprocessor output of the CMSIS-Toolbox, typically the file `.\tmp\1\ac6_linker_script.sct` to `<MyFolder>/Board/<board>`.
-  - In `cdefault.yml` there may be some linker controls that should be reflected in this dialog, for example `--diag_suppress=L6314W`.
+  - The native uVision project manager does not offer the same [linker script management](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/build-overview.md#linker-script-management) as a CMSIS Solution project in VS Code IDE. Therefore follow these steps to configure a scatter file.
```

Comment:
Note that this is a feature of  the CMSIS-Toolbox/csolution rather than the VSCode extension!

Assuming that the user does not modify regions_xxx.h / or ac6_linker_script.sct.src I don't see why not copying the output from the pre-processing step. The user could edit the scatter file rather than preprocessing the file every time.

Reply:
Yes, I know it is a feature of CMSIS-Toolbox, where a standard linker script template + regions.h can be combined and preprocessed to output a real scatter file for the linker.
But is this doc section meant for creating a native µVision project from scratch? If yes, where do you get the "the output from the pre-processing step"? In such a case there is ready-to-use preprocessed scatter file for the µVision project. In other words, there is no such a thing ".\tmp\1\ac6_linker_script.sct"

Reply:
@ReinhardKeil, I think Chen has a valid point here. The Board Layer project does not ship with the pre processed `ac6_linker_script.sct` file as it is only generated in a pre-build step by `cbuild` and uVision does not support this feature natively.

Reply:
Yes, I agree Chen has a valid point.  We take it in as is for now, but the whole section perhaps need more work...